src/xcode/ENA/ENA/Source/Client/HTTP Client/CachingHTTPClient.swift

```diff
@@ -75,7 +75,7 @@ class CachingHTTPClient: AppConfigurationFetching {
 				// serialize config
 				do {
 					let config = try SAP_Internal_V2_ApplicationConfigurationIOS(serializedData: package.bin)
-					let eTag = response.httpResponse.value(forHTTPHeaderField: "ETag")
+					let eTag = response.httpResponse.value(forCaseInsensitiveHeaderField: "ETag")
```

Comment:
I would stick to the old name and 'just make it' case insensitive as the standard behaviour. They were also more meaningful. We only had issues because we use the wrong (case sensitive) way of accessing the fields.

Reply:
`forHTTPHeaderField` is already case-insensitive. From the docs: "The name is case-insensitive."

Reply:
The old name is part of Foundation of iOS 13+, how could I keep the old name and make it work for iOS 12 as well?